src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -243,12 +244,25 @@ class FloatingActionButtonAndPopover extends React.Component {
                               ]
                             : []),
                     ]}
+                    withoutOverlay
+                    anchorRef={this.anchorRef}
                 />
                 <FloatingActionButton
                     accessibilityLabel={this.props.translate('sidebarScreen.fabNewChat')}
                     accessibilityRole="button"
                     isActive={this.state.isCreateMenuActive}
-                    onPress={this.showCreateMenu}
+                    ref={this.anchorRef}
+                    onPress={(e) => {
+                        if (e.nativeEvent.anchorRef && e.nativeEvent.anchorRef.current === this.anchorRef.current) {
```

Comment:
maybe we can extract this to re-usable function - there are a lot of duplication for this code, and imagine one day we will need to make a small change here - and it will be a lot of jumps between files